src/python/pants/backend/python/goals/package_pex_binary_integration_test.py

```diff
@@ -147,12 +148,14 @@ def test_include_sources_avoids_files_targets_warning(
                         name='my-dist',
                         version='1.2.3',
                     ),
+                    interpreter_constraints=["CPython==3.10.*"]
                 )
 
                 pex_binary(
                     dependencies=[':wheel'],
                     entry_point="none",
                     include_sources=False,
+                    interpreter_constraints=["CPython==3.10.*"]
```

Comment:
Sorry missed that this test was failing in the pipeline report.

These tighter constarints avoid
```
ARNING  pants.backend.python.util_rules.pex_cli:pex_cli.py:217 /home/ecsb/.cache/pants/named_caches/pex_root/installed_wheels/f52d67567b65f94aedbde9c10f417df9df1c78a2568cbc7820706a7b3ee6ace2/pex-2.1.159-py2.py3-none-any.whl/pex/bin/pex.py:923: PEXWarning: Could not calculate a targeted shebang for:
cp37-cp37m-manylinux_2_37_x86_64 interpreter at /usr/local/bin/python3.7
cp38-cp38-manylinux_2_37_x86_64 interpreter at /usr/bin/python3.8
cp39-cp39-manylinux_2_37_x86_64 interpreter at /home/ecsb/.cache/pants/named_caches/pex_root/venvs/f2a0eb4c2e2336b7e77df777bff977abf3ca390a/d56abaad3dd7630971f10921587fc3ac5e2cee3c/bin/python3.9
cp39-cp39-manylinux_2_37_x86_64 interpreter at /usr/bin/python3.9

Using shebang: #!/usr/bin/env python3.7
If this is not appropriate, you can specify a custom shebang using the --python-shebang option.
  pex_warnings.warn(
```

which in term allows the "there is no output" assertion in the test to pass.